release/ray_release/byod/byod_vicuna_test.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+# This script is used to build an extra layer on top of the base anyscale/ray image 
+# to run the air_example_vicuna_13b_lightning_deepspeed_finetuning test.
+
+set -exo pipefail
+
+echo "sudo lsblk -f" >> ~/.bashrc
+echo "yes N | sudo mkfs -t ext4 /dev/nvme1n1 || true" >> ~/.bashrc
+echo "mkdir -p /mnt/local_storage" >> ~/.bashrc
+echo "sudo chmod 0777 /mnt/local_storage" >> ~/.bashrc
+echo "sudo mount /dev/nvme1n1 /mnt/local_storage || true" >> ~/.bashrc
+pip3 install torch torchvision torchaudio --index-url https://download.pytorch.org/whl/cu118
+pip3 install "pytorch_lightning==2.0.3" "transformers==4.30.2" "accelerate==0.20.3" "deepspeed==0.9.4"
```

Comment:
why are these pip installed here?

Reply:
I'm not sure, @justinvyu do you know whether the test only works with these pinned versions, or the versions can be more fluid? Thankkks

Reply:
Yeah since the transformers,deepspeed,lightning repos are all constantly changing, so we pin all these libraries to avoid flakiness.

Reply:
ah got you, sound like a good candidate for the shared requirements though, since packages are also pinned there. I'll try that and if that works I'll remove these package pinning here.